services/diversity/src/persistence/models/richness_nos_regions.js

```diff
@@ -0,0 +1,6 @@
+module.exports = bookshelf => (
```

Comment:
This is never used as a model, I think we should delete it

Reply:
richnes_nos_regions.js file was removed and its references in setup file were removed.

Reply:
richness_nos_regios.js deleted.